### PR TITLE
20395: Added ARMGOV (32) to supported clouds and added support for ARMGOV accounts

### PR DESCRIPTION
**Pull request by mlin-aviatrix**

### CODE REVIEW

File: aviatrix/resource_aviatrix_account_test.go [hunk 1]
```diff
@@ -306,6 +355,20 @@ resource "aviatrix_account" "oci" {
 		os.Getenv("OCI_COMPARTMENT_ID"), os.Getenv("OCI_API_KEY_FILEPATH"))
 }
 
+func testAccAccountConfigARMGOV(rInt int) string {
+	return fmt.Sprintf(`
+resource "aviatrix_account" "armgov" {
```
Comment (CyrusJavan) [inline aviatrix/resource_aviatrix_account_test.go]: Line up `=`

File: docs/resources/aviatrix_account.md [hunk 2]
```diff
@@ -68,6 +68,17 @@ resource "aviatrix_account" "temp_acc_oci" {
 }
 ```
 ```hcl
+# Create an Aviatrix Azure Gov Account
+resource "aviatrix_account" "temp_acc_armgov" {
```
Comment (CyrusJavan) [inline docs/resources/aviatrix_account.md]: Line up `=`

**Review by CyrusJavan — DISMISSED** [review]

**Review by zhiqiangjin — DISMISSED** [review]

**Review by shuoliugit — DISMISSED** [review]

File: aviatrix/data_source_aviatrix_account.go [hunk 3]
```diff
@@ -53,6 +53,11 @@ func dataSourceAviatrixAccount() *schema.Resource {
 				Computed:    true,
 				Description: "Azure Subscription ID.",
 			},
+			"armgov_subscription_id": {
```
Comment (zhiqiangjin) [inline aviatrix/data_source_aviatrix_account.go]: Should we use azure_gov_subscription_id?

File: aviatrix/resource_aviatrix_account.go [hunk 4]
```diff
@@ -149,6 +149,30 @@ func resourceAviatrixAccount() *schema.Resource {
 				Sensitive:   true,
 				Description: "OCI API Private Key local file path.",
 			},
+			"armgov_subscription_id": {
```
Comment (zhiqiangjin) [inline aviatrix/resource_aviatrix_account.go]: Same for following four attributes: should we use azure_gov_subscription_id?

File: goaviatrix/account.go [hunk 5]
```diff
@@ -48,6 +48,10 @@ type Account struct {
 	OciUserID                             string `form:"oci_user_id" json:"oci_user_id,omitempty"`
 	OciCompartmentID                      string `form:"oci_compartment_id" json:"oci_compartment_id,omitempty"`
 	OciApiPrivateKeyFilePath              string `form:"oci_api_key_path" json:"oci_api_private_key_filepath,omitempty"`
+	ArmgovSubscriptionId                  string `form:"azure_gov_subscription_id,omitempty" json:"arm_gov_subscription_id,omitempty"`
```
Comment (zhiqiangjin) [inline goaviatrix/account.go]: Same here.

File: goaviatrix/const.go [hunk 6]
```diff
@@ -9,6 +9,7 @@ const (
 	GCP    = 4
 	AZURE  = 8
 	OCI    = 16
+	ARMGOV = 32
```
Comment (zhiqiangjin) [inline goaviatrix/const.go]: Maybe: AzureGOV

**Review by zhiqiangjin — CHANGES_REQUESTED** [review]

**Review by zhiqiangjin — APPROVED** [review]

**Pull request merged**